### ISSUE TITLE
Enhancement Request: Enable usage of proposed APIs by CodeTogether

### ISSUE THREAD

**Issue by toddewilliams** [arduino/arduino-ide#930]

**Comment by toddewilliams** [comment]
@gmacario Yes! Apologies for posting on the wrong repo and thank you very much for kindly noting it and transferring it to where it belonged. I truly appreciate it!

**Comment by gmacario** [comment]
> I truly appreciate it!

You are welcome 😉

**Comment by toddewilliams** [comment]
CodeTogether does not currently syntax highlight Arduino sketch files for remote participants, but we’ll remedy that with [this enhancement request for CodeTogether](https://github.com/Genuitec/CodeTogether/issues/364).

If there are file types other than .ino that are unique to Arduino development, I’d appreciate any information on those as comments to the CodeTogether enhancement above.
